doc/configuration.md

```diff
@@ -78,3 +81,7 @@ This method allows you to set the default endpoint to use. This means, that when
 ## 7: Chain
 
 All methods return `this`, allowing you to chain the calls.
+
+## 8: Set default base URL
+
+All endpoints registered after this call will have the passed in URL prepended by default, rather than the current host. 
```

Comment:
Yes, you're correct. However, if a Endpoint A is registered before `setBaseDefaultUrl()`, then Endpoint A will use the current host url.

For example:
```
config.registerEndpoint('api'); // currenthost/api
config.setDefaultUrl('api.example.com');
config.registerEndpoint('example'); // api.example.com/example
```